buster/documents_manager/service.py

```diff
@@ -76,3 +80,14 @@ def delete_source(self, source: str) -> tuple[int, int]:
         self.index.delete(filter={"source": source}, namespace=self.namespace)
 
         return source_deleted, documents_deleted
+
+    def drop_db(self):
```

Comment:
can we add an `input()` where user has to explicitly agree to a destructive action? We should make it clear that this is not revertable

Reply:
done